website/docs/r/pod.html.markdown

```diff
@@ -328,7 +332,7 @@ For more info see [Kubernetes reference](http://kubernetes.io/docs/user-guide/la
 * `lifecycle` - (Optional) Actions that the management system should take in response to container lifecycle events
 * `liveness_probe` - (Optional) Periodic probe of container liveness. Container will be restarted if the probe fails. Cannot be updated. For more info see [Kubernetes reference](http://kubernetes.io/docs/user-guide/pod-states#container-probes)
 * `name` - (Required) Name of the container specified as a DNS_LABEL. Each container in a pod must have a unique name (DNS_LABEL). Cannot be updated.
-* `port` - (Optional) List of ports to expose from the container. Exposing a port here gives the system additional information about the network connections a container uses, but is primarily informational. Not specifying a port here DOES NOT prevent that port from being exposed. Any port which is listening on the default "0.0.0.0" address inside a container will be accessible from the network. Cannot be updated.
+* `port` - (Optional) Ports to expose from the container. Exposing a port here gives the system additional information about the network connections a container uses, but is primarily informational. Not specifying a port here DOES NOT prevent that port from being exposed. Any port which is listening on the default "0.0.0.0" address inside a container will be accessible from the network. Cannot be updated.
```

Comment:
This should probably say it is a block, which is documented below: https://github.com/terraform-providers/terraform-provider-kubernetes/blame/e3953ef453a0940fd5dfd64c11cdb7b46423a11f/website/docs/r/pod.html.markdown#L632-L640, and that there can be multiple occurrences.

Maybe a mix of what's already done for `dns_config` `option`:
https://github.com/terraform-providers/terraform-provider-kubernetes/blame/e3953ef453a0940fd5dfd64c11cdb7b46423a11f/website/docs/r/pod.html.markdown#L426

and `env`: 

https://github.com/terraform-providers/terraform-provider-kubernetes/blame/e3953ef453a0940fd5dfd64c11cdb7b46423a11f/website/docs/r/pod.html.markdown#L328

Reply:
+1 on what @pdecat mentioned. It should reference the expanded block documentation.

You are right that this is not exactly expressed as a list in TF configuration. However, as you mentioned, it translates to a list of ports and relates to a list on the Kubernetes API docs. So I think we should make sure this is somehow expressed in here as well.
I think mentioning that "It can be expressed multiple times." should be sufficient for that. What do you think?

Reply:
@pdecat @alexsomesan I think I captured what you both were saying in https://github.com/terraform-providers/terraform-provider-kubernetes/pull/698/commits/6b04cea8b750806df27031d5172a5e611805d094. I was a little confused whether I should stick with the text "May be used multiple times" that's used elsewhere, or I should say "it can be expressed multiple times" as you mentioned @alexsomesan. Also, I didn't quite know how to say that it gets translated to a list in the Kubernetes API docs.

Let me know what you both think!

Reply:
Way better :+1: !